DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1003 @@ Live discussion in #document-authoring on Slack.
+</div>
+
+<div class="example">
+Below is an example of the explicit addressing mode used in a scenario where different [=media segments=] have different durations (e.g. due to encoder limitations).
+
+The example defines a sequence of 11 [=media segments=] starting at position 120 on the [=sample timeline=] and lasting for a total of 95520 units at a timescale of 1000 units per second (which results in 95.52 seconds of data). The [=period=] starts at position 810, which is within the first [=media segment=], found at the relative URL `video/120.m4s`. The fifth [=media segment=] repeats once, resulting in a sixth [=media segment=] with the same duration.
+
+<xmp highlight="xml">
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011">
+	<Period>
+		<AdaptationSet>
+			<Representation>
+				<SegmentTemplate timescale="1000" presentationTimeOffset="810"
+						media="video/$Time$.m4s" initialization="video/init.mp4">
+					<SegmentTimeline>
+						<S t="120" d="8520"/>
+						<S d="8640"/>
+						<S d="8600"/>
+						<S d="8680"/>
+						<S d="9360" r="1"/>
+						<S d="8480"/>
+						<S d="9080"/>
+						<S d="6440"/>
+						<S d="10000"/>
+						<S d="8360"/>
+					</SegmentTimeline>
+				</SegmentTemplate>
+			</Representation>
+		</AdaptationSet>
+	</Period>
+</MPD>
+</xmp>
+
+Parts of the MPD structure that are not relevant for this chapter have been omitted - this is not a fully functional MPD file.
+</div>
+
+#### Moving the period start point (explicit addressing) #### {#timing-addressing-explicit-startpoint}
+
+When splitting periods in two or performing other types of editorial timing adjustments, a service might want to start a period at a point after the "natural" start point of the [=representations=] within.
+
+For [=representations=] that use [=explicit addressing=], perform the following adjustments to set a new [=period=] start point:
+
+<div class="algorithm">
+
+1. Update `SegmentTemplate@presentationTimeOffset` to indicate the desired start point on the [=sample timeline=].
+1. Update `Period@duration` to match the new duration.
+1. Remove any unnecessary segment references.
+
+</div>
+
+Note: See [[#timing-representation]] and [[#timing-mpd-updates-remove-content]] to understand the constraints that apply to [=segment reference=] removal.
+
+### Simple addressing ### {#timing-addressing-simple}
+
+A representation that uses <dfn>simple addressing</dfn> consists of a set of [=media segments=] accessed via URLs constructed using a template defined in the MPD, with the nominal time span covered by each [=media segment=] described in the MPD.
+
+Advisement: Simple addressing defines the nominal time span of each [=media segment=] in the MPD. The true time span covered by samples within the [=media segment=] can be slightly different than the nominal time span. See [[#timing-addressing-inaccuracy]].
+
+Clauses in section only apply to [=representations=] that use simple addressing.
+
+<figure>
+	<img src="Images/Timing/SimpleAddressing.png" />
```

Comment:
This figure contains `$Time$` in the illustration, should it be changed to `$Number$` for simple addressing?

Reply:
Well, `$Time$` is allowed for simple addressing, so it is a valid scenario for simple addressing. Maybe I can put both modes on the diagram? I will try.

This makes me realize that nowhere does this chapter mention `$Time$` or `$Number$`. I will add a mention.